packages/@aws-cdk-testing/framework-integ/test/aws-kms/test/integ.alias-from-alias-name.ts

```diff
@@ -0,0 +1,26 @@
+import { App, Stack } from 'aws-cdk-lib';
+import { Alias } from 'aws-cdk-lib/aws-kms';
+import { IntegTest } from '@aws-cdk/integ-tests-alpha';
+import { AccountRootPrincipal, Role } from 'aws-cdk-lib/aws-iam';
+import * as cxapi from 'aws-cdk-lib/cx-api';
+
+const app = new App({
+  context: { [cxapi.KMS_APPLY_IMPORTED_ALIAS_PERMISSIONS_TO_PRINCIPAL]: true },
+});
+const stack = new Stack(app, 'aws-cdk-kms');
+const alias = Alias.fromAliasName(stack, 'alias', 'alias/MyKey');
+
+const role = new Role(stack, 'Role', {
+  assumedBy: new AccountRootPrincipal(),
```

Comment:
These permissions are too broad for a [scanner we have on integ test templates](https://github.com/aws/aws-cdk/actions/runs/15612314387/job/44034398202?pr=34237), lets try to change, and see if it helps...
```suggestion
  assumedBy: new ServicePrincipal('lambda.amazonaws.com'),
```

Reply:
Thanks for looking into it. This is a funny finding for an integration test. Commited it. Let's see.